http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -1462,14 +1462,13 @@ class GKEInterpreter[F[_]](
     gsa: WorkbenchEmail,
     customEnvironmentVariables: Map[String, String]
   )(implicit ev: Ask[F, AppContext]): F[Unit] = {
-    // TODO: Use the chart from the database instead of re-looking it up in config:
```

Comment:
leo loads all config files into a global object upfront...so it's cheaper to read config values than doing a DB lookup